cosim-cli.md

```diff
@@ -115,22 +115,6 @@ variables:
     variability: continuous
 ```
 
-```
-> cosim inspect fmu-proxy://10.0.0.65:9090?url=http://simple-models.com/circle_area.fmu
```

Comment:
Can this be replaced instead of removed?
Like: `> cosim inspect proxyfmu://localhost?file=../circle_area.fmu`

Or maybe this example doesn't give much sense as url is not supported...?

Reply:
Yeah, it does not make sense as the FMU is local on your computer anyway.

Reply:
ok